package/base-files/files/etc/init.d/led

```diff
@@ -49,6 +55,14 @@ load_led() {
 	[ -e /sys/class/leds/${sysfs}/brightness ] && {
 		echo "setting up led ${name}"
 
+		[ "${uci_brightness}" -lt 0 ] || [ "${uci_brightness}" -gt "${uci_max_brightness}" ] &&
+			uci_brightness="${uci_max_brightness}"
+
+		# Convert UCI brightness by doing a cross product (uci_brightness / uci_max_brightness) = (x / sysfs_max_brightness).
+		# The cross product is rounded up to avoid turning off the led even if there is a valid positive brightness configured.
+		read sysfs_max_brightness < "/sys/class/leds/${sysfs}/max_brightness"
```

Comment:
Consider revising this to e.g. 
```sh
		sysfs_brightness=$(( ( $uci_brightness * $(cat "/sys/class/leds/${sysfs}/max_brightness") + $uci_max_brightness - 1 ) / $uci_max_brightness ))
```

Consider also using `expr` so as to coerce strings to numbers, to prevent possible errors from that vector.